doc/1.4/language.md

```diff
@@ -2821,28 +2854,84 @@ used in expression-statements.
 
 DML adds the following statements:
 
+### Assignment Statements
+<pre>
+<em>target1</em> [= <em>target2</em> = <em>...</em>] = <em>initializer</em>;
+(<em>target1</em>, <em>target2</em>, ...) = <em>initializer</em>;
+</pre>
+
+Assign values to targets according to an initializer. Unlike C, assignments are
+not expressions, and the right-hand side can be any initializer -- such as
+compound initializers (<tt>{<em>...</em>}</tt>) for struct-like types.
+
+The first form is chaining assignments. The initializer is executed once and
+the value it evaluates to is assigned to each target.
+
+The second form is multiple simultaneous assignment. The initializer describes
+multiple values -- one for each target. This can be done either through:
+* Providing an initializer for each target through tuple syntax, e.g.:
+```
+(a, i) = (false, 4);
+```
+* Performing a method call where each target is a return value recipient, e.g.:
+```
+method m() -> (bool, int) {
+    ...
+}
+```
+```
+(a, i) = m();
+```
+
+Targets are updated simultaneously, meaning it's possible to e.g. swap the
+contents of variables through the following:
+```
+(a, b) = (b, a)
+```
```

Comment:
You could consider a note with a cross-reference to the syntactically related multi-value return statements and variable declarations, if that can be done without disturbing the flow.

Reply:
No, I can't make it fit. I knit them together in new changes to the method calls section, though.